### DISCUSSION TITLE
Flex and Composer

### DISCUSSION THREAD

**Discussion by gopikrish19** [symfony/symfony#44113] [Q&A]
Can any one explain how Symfony/flex works with composer, When symfony/Flex starts working(Symfony operations getting perform), is composer calling Flex during composer require or install

**Comment by mbabker** [comment]
Flex is a Composer plugin.  So Flex hooks into Composer using Composer's events to handle its business.

**Reply by gopikrish19** [reply]
Thank you

**Reply by gopikrish19** [reply]
Can we create one new recipe for a new custom bundle ? , if yes, how can i host that into flex server ? 
And, How flex make that configs for the new custom bundle?

**Reply by ahundiak** [reply]
I have not tried it myself but you can start here: https://symfony.com/blog/symfony-flex-is-going-serverless#custom-recipes
